src/base/utils/net.cpp

```diff
@@ -26,9 +26,12 @@
  * exception statement from your version.
  */
 
+#include "base/preferences.h"
```

Comment:
Would you rather the `QStringList` of subnets was explicitly passed to the function as an argument?